lib/models/NotificationConfiguration.js

```diff
@@ -289,21 +289,23 @@ class NotificationConfiguration {
      */
     static validateConfig(config) {
         assert.strictEqual(typeof config, 'object');
-        assert.strictEqual(Array.isArray(config.queueConfig), true);
-        config.queueConfig.forEach(q => {
-            const { events, queueArn, filterRules, id } = q;
-            assert.strictEqual(Array.isArray(events), true);
-            events.forEach(e => assert.strictEqual(typeof e, 'string'));
-            assert.strictEqual(typeof queueArn, 'string');
-            if (filterRules) {
-                assert.strictEqual(Array.isArray(filterRules), true);
-                filterRules.forEach(f => {
-                    assert.strictEqual(typeof f.name, 'string');
-                    assert.strictEqual(typeof f.value, 'string');
-                });
-            }
-            assert.strictEqual(typeof id, 'string');
-        });
+        if (config.queueConfig) {
```

Comment:
Regardless of my other comment, I would turn this into a guard clause and early return over the extra tab.

Reply:
the way that function is used doesn't allow for it to have a return, so there's not really a way to make it a guard clause

Reply:
Does this not work?
```javascript
if (!config.queueConfig) {
    return
}
```

Reply:
i mean yeah, but it's extra lines vs an extra tab. i prefer an extra tab. we can leave it up to jonathan if he wants to enforce a style

Reply:
Personally I tend to prefer `return` for error cases to return early from the whole processing, and use a block when the flow logic becomes more complex. But I don't really mind in the end as long as the indentation level does not go up too much.

Reply:
For me when writing javascript, especially when using callbacks, the reduced indentation levels is usually preferred over the one saved line. The opening and close brace are required regardless, it is just the return line that is added with guard clauses.